tools/genconversion/conversion.go

```diff
@@ -13,50 +13,61 @@ import (
 	"k8s.io/kubernetes/cmd/libs/go2idl/args"
 	"k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators"
 	"k8s.io/kubernetes/cmd/libs/go2idl/generator"
+	"k8s.io/kubernetes/pkg/util/sets"
 )
 
 func main() {
 	arguments := args.Default()
 
-	// Override defaults. These are Kubernetes specific input locations.
-	arguments.InputDirs = []string{
+	// These are the packages we expect generated conversions for
+	expectedPackages := sets.NewString(
+		"github.com/openshift/origin/pkg/authorization/api/v1",
```

Comment:
I wonder if we can make this a little bit more intelligent by scanning the origin/pkg/* recursively and looking for `/api/v1` folders... Then check if the folder has "doc.go" file (give clues if it does not).